packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.test.ts

```diff
@@ -122,6 +118,23 @@ const getTransactionMockResponse = {
   }
 }
 
+const getNetworkFeesMockedResponse = {
+  data: {
+    fast: {
+      blocksUntilConfirmation: 1,
+      satsPerKiloByte: 1024
```

Comment:
what's up with sats per kilo? txs can be significantly less than 1k

Reply:
thats how it comes back from unchained

Reply:
Value being returned from estimatesmartfee call: https://bitcoincore.org/en/doc/0.21.0/rpc/util/estimatesmartfee/